SocialCareCaseViewerApi.Tests/V1/Controllers/MashReferral/CreateMashReferralTests.cs

```diff
@@ -0,0 +1,34 @@
+using Moq;
+using NUnit.Framework;
+using SocialCareCaseViewerApi.Tests.V1.Helpers;
+using SocialCareCaseViewerApi.V1.Boundary.Requests;
+using SocialCareCaseViewerApi.V1.Controllers;
+using SocialCareCaseViewerApi.V1.UseCase.Interfaces;
+
+namespace SocialCareCaseViewerApi.Tests.V1.Controllers.MashReferral
+{
+    [TestFixture]
+    public class CreateMashReferralTests
+    {
+        private Mock<IMashReferralUseCase> _mashReferralUseCase;
+        private MashReferralController _mashReferralController;
+
+        [SetUp]
+        public void Setup()
+        {
+            _mashReferralUseCase = new Mock<IMashReferralUseCase>();
+            _mashReferralController = new MashReferralController(_mashReferralUseCase.Object);
+        }
+
+        [Test]
+        public void CallsMashReferralUseCaseToInsertNewReferral()
+        {
+            _mashReferralUseCase.Setup(x => x.CreateNewMashReferral(It.IsAny<CreateReferralRequest>()));
+            var request = TestHelpers.CreateNewMashReferralRequest();
+
+            _mashReferralController.CreateNewContact(request);
+
+            _mashReferralUseCase.Verify(x => x.CreateNewMashReferral(request));
+        }
```

Comment:
Possibly an additional test to show the ObjectResponse that is returned from the Controller

Reply:
Done in 073802a